packages/css-dev-tools/postcssPluginConfig.js

```diff
@@ -6,10 +6,19 @@ const base64 = require('postcss-base64')
 const reporter = require('postcss-reporter')
 const cssnano = require('cssnano')
 const mqpackerondemand = require('@mozaic-ds/postcss-media-queries-packer-on-demand')
+const cssbyebye = require('css-byebye')
 const CM = require('@mozaic-ds/configuration-manager')
 
 const MOZAIC_ENV = process.env.MOZAIC_ENV
 
+const rulesToRemove = [
```

Comment:
Don't use this to remove css rules, use the variable you created and an explicit test in the source code

Reply:
Fixed by using a custom PostCSS plugin to inject content in PostCSS